rpc/rpc_machine.c

```diff
@@ -468,12 +469,13 @@ static int rpc_tm_setup(struct m0_net_transfer_mc *tm,
 	if (rc < 0)
 		return M0_ERR_INFO(rc, "TM initialization");
 
+	if (m0_streq(net_dom->nd_xprt->nx_name, "lnet"))
+		max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);
+	else
+		max_msgs_size = 1;
+
```

Comment:
I think you can have :
max_msgs_size = m0_rpc_max_msg_size(net_dom, msg_size);

because it already has been handled.